docs/campaigns/campaign_builder.rst

```diff
@@ -251,3 +251,31 @@ Actions and Decisions in Mautic require a :doc:`cron job</configuration/cron_job
    php /path/to/mautic/bin/console mautic:campaigns:trigger
 
 If you want to execute the command at different intervals for specific Campaigns, you can pass the ``--campaign-id=ID`` argument to the command.
+
+.. vale off
+
+Cloning Campaign events
+-----------------------
+
+.. vale on
+
+Mautic's Campaign builder now includes a feature that allows Users to clone (copy and paste) Campaign events, making it easier to replicate complex workflows or reuse specific actions, decisions, or conditions across different Campaigns. This feature supports cloning events within the same Campaign as well as between different Campaigns.
```

Comment:
```suggestion
Since Mautic 5.1, the Campaign builder includes a feature that allows Users to clone - copy and paste - Campaign events, making it easier to replicate complex workflows or reuse specific actions, decisions, or conditions across different Campaigns. This feature supports cloning events within the same Campaign as well as between different Campaigns.
```

Reply:
Add a version reference to make it clearer when this was introduced.